### PR TITLE
feat(cxl-ui): cxl-institute-pricing-table

### PR DESCRIPTION
**Pull request by paulkirspuu**

### CODE REVIEW

**Comment by lkraav** [comment]
Commit message is missing "fix #31" for automation triggers.

**Comment by lkraav** [comment]
@paulkirspuu this PR could use a checklist of "road to success" actions needed. What is solved, what is unsolved.

File: packages/cxl-ui/package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
     "@pika/plugin-build-web": "^0.8.1",
     "@pika/plugin-bundle-web": "^0.8.1",
     "@pika/plugin-standard-pkg": "^0.8.1",
+    "@polymer/iron-pages": "^3.0.1",
+    "@polymer/paper-toggle-button": "^3.0.1",
```
Comment (lkraav) [inline packages/cxl-ui/package.json]: Corresponding `yarn.lock` is uncommitted.

**Comment by pawelkmpt** [comment]
@paulkirspuu @lkraav it's old. can we close it?

**Comment by pawelkmpt** [comment]
Obsolete

**Pull request closed without merging**